target/linux/rtl838x/files-5.4/drivers/net/dsa/rtl838x.h

```diff
@@ -0,0 +1,190 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#ifndef _RTL838X_H
+#define _RTL838X_H
+
+#include <net/dsa.h>
+
+// #define RTL838X_SW_BASE ((volatile void *)0xBB000000)
```

Comment:
ERROR: do not use C99 // comments

Can be removed?

Reply:
Will remove.